qcfractal/interface/collections/dataset.py

```diff
@@ -700,7 +700,7 @@ def _get_native_values(
         DataFrame
             A DataFrame of the queried parameters
         """
-        au_units = {"energy": "hartree", "gradient": "hartree/bohr", "hessian": "hartree/bohr**2"}
+        au_units = {"energy": "hartree", "gradient": "hartree/bohr", "hessian": "hartree/bohr**2","properties","hartree"}
```

Comment:
Needs to be `"properties": "hartree"`

Reply:
Yeah sure!  I'm guessing you want me to add the test to test_dataset.py